devops/openshift/api-deploy.yml

```diff
@@ -225,18 +225,30 @@ objects:
       to:
         kind: Service
         name: ${NAME}
-  - apiVersion: autoscaling/v1
+  - apiVersion: autoscaling/v2
     kind: HorizontalPodAutoscaler
     metadata:
       name: ${NAME}-hpa
     spec:
       scaleTargetRef:
-        apiVersion: apps/v1
-        kind: ReplicationController
-        name: ${NAME}-rc
+        apiVersion: apps.openshift.io/v1
+        kind: DeploymentConfig
+        name: ${NAME}
       minReplicas:  "${{REPLICAS}}"
       maxReplicas: 10
-      targetCPUUtilizationPercentage: 80
+      metrics:
+          - type: Resource
+            resource:
+              name: cpu
+              target:
+                type: AverageValue
+                averageValue: '0.18'
```

Comment:
Asking for my clarification, how did we arrive at this average value?

Reply:
90% of the CPU limit